packages/replay-next/src/suspense/FrameStepsCache.ts

```diff
@@ -9,7 +9,7 @@ import { sourcesByIdCache } from "./SourcesCache";
 
 function getKey([client, pauseId, frameId]: [ReplayClientInterface, PauseId, FrameId]) {
   const focusWindow = client.getCurrentFocusWindow();
-  return `${pauseId}:${frameId}` + (focusWindow ? `:${focusWindow.begin}-${focusWindow.end}` : "");
+  return `${pauseId}:${frameId}` + (focusWindow ? `:${focusWindow.begin.point}-${focusWindow.end.point}` : "");
```

Comment:
`TimeStampedPoint` has `.point` and `.time`. From what I understand they both describe the same thing - just in a different format. So we can use one of them, please correct me if I'm wrong about this one

Reply:
Yeah. Point is more precise so using that seems right.